packages/cannoli-core/src/nl-generator/compiler.ts

```
@@ -0,0 +1,531 @@
+					label: edge.label || "config",
+				};
+
+			case "field":
+				return {
+					...baseEdge,
+					color: "6", // Purple
+					label: edge.label || "field",
+				};
+
+			case "choice":
+				let choiceLabel = edge.label || "option";
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)